zebra/interface.c

```diff
@@ -4131,70 +4131,26 @@ DEFPY_YANG(no_link_params_admin_grp, no_link_params_admin_grp_cmd,
 }
 
 /* RFC5392 & RFC5316: INTER-AS */
-DEFUN (link_params_inter_as,
+DEFPY_YANG (link_params_inter_as,
        link_params_inter_as_cmd,
-       "neighbor A.B.C.D as (1-4294967295)",
+       "[no] neighbor ![A.B.C.D$ip as (1-4294967295)$as]",
```

Comment:
It would be nice to use the built in token type `ASNUM` which maps to `<A.B|(1-4294967295)>`; however, which returns an `as_t`, but  you'd then have to convet that to a string b/c `as_str` might be the dotted format.

BTW the `$as` is fine, but redundant. By default the preceding keyword `as` will name the variable.

Reply:
Can be done separately later if necessary. I didn't try to improve any commands in this PR, except for bug-fixes.